logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationRepository.kt

```diff
@@ -222,7 +238,9 @@ internal class ConversationDataSource internal constructor(
         selfUserTeamId: String?,
         originatedFromEvent: Boolean,
     ) = wrapStorageRequest {
-        val conversationEntities = conversations.map { conversationResponse ->
+        val conversationEntities = conversations
+            .filter { !(it.type == ConversationResponse.Type.GLOBAL_TEAM && it.protocol == ConvProtocol.PROTEUS) }
```

Comment:
Working around a bug in the backend where the MLS exclusive team conversation is returned as a proteus conversation.